tensorflow/core/lib/strings/numbers_test.cc

```diff
@@ -278,6 +279,30 @@ TEST(safe_strtof, Float) {
   EXPECT_EQ(-42.0f, result);
 
```

Comment:
maybe test case of leading and trailing whitespace as well

Reply:
Sure, I'll do it. Thank you.